File: dts-dorredlich.client/src/models/customerModel.js
export class CustomerModel {
 
    constructor(customerId, customerName, requestedTime, createdAt = null, id = 0) {
      this.id = id;
      this.customerId = customerId;
      this.customerName = customerName;
      this.requestedTime = requestedTime;
      this.createdAt = createdAt; 
    }

}

File: Data/Models/CustomerModel.cs

namespace Data.Models
{
    public class CustomerModel
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string? CustomerName { get; set; }
        public DateTime RequestedTime { get; set; }
        public DateTime? CreatedAt { get; set; }
    }
}
